src/cascadia/TerminalApp/TerminalPage.h

```diff
@@ -152,7 +152,7 @@ namespace winrt::TerminalApp::implementation
         void _PasteText();
         static fire_and_forget PasteFromClipboard(winrt::Microsoft::Terminal::TerminalControl::PasteFromClipboardEventArgs eventArgs);
 
-        fire_and_forget _LaunchSettings(const bool openDefaults);
+        fire_and_forget _LaunchSettings(const winrt::TerminalApp::SettingsTarget target = winrt::TerminalApp::SettingsTarget::SettingsFile);
```

Comment:
nit: do we really need the default value for this param? I'd think that the default value should be elsewhere, and this method would _require_ the caller to specify. At the same time, I don't care that much.